[2025-02-18T06:42:31.927+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T06:42:31.955+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.extract manual__2025-02-18T06:42:28.134535+00:00 [queued]>
[2025-02-18T06:42:31.976+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.extract manual__2025-02-18T06:42:28.134535+00:00 [queued]>
[2025-02-18T06:42:31.978+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-18T06:42:32.005+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract> on 2025-02-18 06:42:28.134535+00:00
[2025-02-18T06:42:32.020+0000] {standard_task_runner.py:72} INFO - Started process 3878 to run task
[2025-02-18T06:42:32.026+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'extract', 'manual__2025-02-18T06:42:28.134535+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/DAG-fraud.py', '--cfg-path', '/tmp/tmp5kf6xr5w']
[2025-02-18T06:42:32.030+0000] {standard_task_runner.py:105} INFO - Job 158: Subtask extract
[2025-02-18T06:42:32.114+0000] {task_command.py:467} INFO - Running <TaskInstance: ml_pipeline.extract manual__2025-02-18T06:42:28.134535+00:00 [running]> on host 8538735272b1
[2025-02-18T06:42:32.256+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='group1' AIRFLOW_CTX_DAG_ID='ml_pipeline' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T06:42:28.134535+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-18T06:42:28.134535+00:00'
[2025-02-18T06:42:32.260+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T06:42:32.300+0000] {base.py:84} INFO - Retrieving connection 'postgres_***'
[2025-02-18T06:42:32.325+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.9/site-packages/pandas/io/sql.py:761 UserWarning: pandas only support SQLAlchemy connectable(engine/connection) ordatabase string URI or sqlite3 DBAPI2 connectionother DBAPI2 objects are not tested, please consider using SQLAlchemy
[2025-02-18T06:42:32.876+0000] {DAG-fraud.py:42} INFO - Data extraction successful.
[2025-02-18T06:42:32.879+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T06:42:32.895+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T06:42:32.897+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=extract, run_id=manual__2025-02-18T06:42:28.134535+00:00, execution_date=20250218T064228, start_date=20250218T064231, end_date=20250218T064232
[2025-02-18T06:42:32.971+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T06:42:33.027+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-18T06:42:33.031+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
